doc/cephfs/add-remove-mds.rst

```diff
@@ -1,3 +1,10 @@
+.. note::
+   It is highly recommended to use `cephadm`_ or another Ceph orchestrator for
+   setting up the ceph cluster and thus the MDS(s). Use this approach only if
+   you are setting up the ceph cluster manually. If one still intends to use
```

Comment:
s/ceph/Ceph/

Reply:
Sorry, I didn't understand, can you please elaborate on it?

Reply:
Replace "ceph" with "Ceph"